### ISSUE TITLE
Lodash Security Advisory

### ISSUE THREAD

**Issue by 0xcaff** [Asana/node-asana#167]
`npm audit` is complaining about the version of lodash node-asana uses.

```

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ asana                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ asana > lodash                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 5602 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by jjschnei** [comment]
Thanks for pointing this out. We'll look into it.

**Comment by praecipula** [comment]
Thanks for letting us know this was somewhat urgent! It should be fixed now as of v0.16.6
